### PR TITLE
Module 1

### PR DESCRIPTION
**Pull request by pldvd**
Solutions for Module 1.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 node_modules
 .env
-test.ts
\ No newline at end of file
+test.ts
```
Comment (danielcsato) [inline .gitignore:4]: Is this needed?

File: module-1/task2.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+const path = require('path');
+const csv = require('csvtojson');
+const { createReadStream, createWriteStream } = require('fs');
+const { pipeline } = require('node:stream/promises');
+
+*/
+
+/*
+ES MODULES
+*/
```
Comment (danielcsato) [inline module-1/task2.ts]: feel free to remove commonjs solution

File: package-lock.json
Comment (danielcsato) [inline package-lock.json:1]: include this file in `.gitignore` please

**Review by danielcsato — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (pldvd) [reply .gitignore:4]: I like coding along with the lessons, and I thought it does no harm if I have this on my local machine only. I think coding along with the lessons helps learning, and this file serves that purpose. So I like having it there. In your opinion, does it violate any principles?

Comment (danielcsato) [reply .gitignore:4]: Nah feel free to leave it I was curious if it was intended or not

**Pull request merged**